pandas/plotting/_core.py

```diff
@@ -1089,14 +1089,20 @@ def __call__(self, *args, **kwargs):
     @Substitution(kind="line")
     @Appender(_bar_or_line_doc)
     def line(
-        self, x: Hashable | None = None, y: Hashable | None = None, **kwargs
+        self,
+        x: Hashable | None = None,
+        y: Hashable | None = None,
+        color: str | Sequence[str] | dict | None = None,
+        **kwargs,
     ) -> PlotAccessor:
         """
         Plot Series or DataFrame as lines.
 
         This function is useful to plot lines using DataFrame's values
         as coordinates.
         """
+        if color is not None:
```

Comment:
Instead can you just specify `return self(..., color=color, **kwargs)`? (and likewise below)

Reply:
This would be prettier, but calling `self(..., color=None, colormap=foo)` raises `UserWarnings` for simultaneous usage of conflicting args, and the function defaults to use `color=None`, ignoring the value of `colormap`.

Similar situation for `y` in the pieplot, which conflicts with the usage of `subplots` depending on their respective values.